m1well.Utils/src/cleanup.js

```diff
@@ -0,0 +1,177 @@
+// @flow
```

Comment:
Not a big deal but "cleanup" seems like a strange filename for sorter to be in. I wouldn't think to look there for task sorting code. So I had to trace it from index.

Reply:
because there are functions which "cleans up" a note ;)